### PR TITLE
TIQR-300 removed .strings and removed logic for showing the option dialog

### PR DESCRIPTION
**Pull request by bbo-git**
I removed the logic that shows the dialog as well as removed the strings

I never encountered code where you have access to this setting....

### CODE REVIEW

File: Sources/TiqrCoreObjC/Classes/TiqrCoreManager.m [hunk 1]
```diff
@@ -68,14 +68,14 @@ + (instancetype)sharedInstance {
 
 - (UINavigationController * _Nonnull)startWithOptions:(NSDictionary * _Nullable)launchOptions {
 
-	NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];	
-	BOOL showInstructions = 
-        [defaults objectForKey:@"show_instructions_preference"] == nil || 
-        [defaults boolForKey:@"show_instructions_preference"];		
+//	NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
```
Comment (dzolnai) [inline Sources/TiqrCoreObjC/Classes/TiqrCoreManager.m]: Why don't we remove this?

**Review by dzolnai — CHANGES_REQUESTED** [review]

File: Sources/TiqrCoreObjC/Classes/TiqrCoreManager.m (hunk 1 above)
Comment (bbo-git) [reply Sources/TiqrCoreObjC/Classes/TiqrCoreManager.m]: Ah yes, my bad

**Comment by bbo-git** [comment]
https://user-images.githubusercontent.com/73382903/201093948-5ee7a1e4-5d70-4760-9b47-c76167b30a13.mov

strange UI glitch

**Comment by dzolnai** [comment]
It seems that it still wants to starts scanning for a split second

**Review by dzolnai — APPROVED** [review]

**Pull request merged**